Electron/Core/RoveProtocol/Rovecomm.ts

```diff
@@ -4,6 +4,87 @@ const dgram = require("dgram")
 const net = require("net")
 const EventEmitter = require("events")
 
+function decodePacket(
+  size: number,
+  dataLength: number,
+  data: Uint8Array
+): number[] {
+  const retArray = []
+  let i = 0
+  let j = 0
+  switch (size) {
+    case 1:
+      while (i < dataLength) {
+        retArray.push(Number(data[i]))
```

Comment:
We probably need to consider what exactly to return as, Number, BigInt, and array of uint/int 8/16/32?

Reply:
I think standardizing on either Number or BigInt is probably the best way to go to provide a sufficiently abstracted interface for other components and not require them to know which data type they're receiving.